exchanges/exchange.go

```diff
@@ -214,7 +214,7 @@ func (e *Base) GetAssetTypes() asset.Items {
 func (e *Base) GetPairAssetType(c currency.Pair) (asset.Item, error) {
 	assetTypes := e.GetAssetTypes()
 	for i := range assetTypes {
-		if e.GetEnabledPairs(assetTypes[i]).Contains(c, true) {
+		if e.GetAvailablePairs(assetTypes[i]).Contains(c, true) {
```

Comment:
for example if we were to get open orders or something for all pairs it could cause issues if not all pair types r enabled so changed to available pairs so things wont error out